pkg/util/util.go

```diff
@@ -36,12 +37,43 @@ import (
 const (
 	// MiB - MebiByte size
 	MiB = 1024 * 1024
+	GiB = MiB * 1024
 )
 
-// RoundUpToMiB rounds up given quantity upto chunks of MiB
-func RoundUpToMiB(size int64) int64 {
+// RoundOffVolSize rounds up given quantity upto chunks of MiB/GiB
+func RoundOffVolSize(size int64) int64 {
```

Comment:
RoundOffVolSize could call RoundOffBytes and convert it into number of MiBs and return?

Reply:
yes  this can be done, updated now